### PR TITLE
Add unit test for ClassSetReader

### PR DESCRIPTION
**Pull request by stephanenicolas**

### CODE REVIEW

File: src/test/kotlin/com/autonomousapps/internal/ClassSetReaderTest.kt [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+import kotlin.test.Test
+import kotlin.test.assertTrue
+
+class ClassSetReaderTest {
+
+    @get:Rule val tempFolder = TemporaryFolder()
+
+    private val shelter = SeattleShelter()
+
+    @Test fun `jar file analysis is correct`() {
```
Comment (autonomousapps) [inline src/test/kotlin/com/autonomousapps/internal/ClassSetReaderTest.kt]: I think a copy-paste error is here and elsewhere in this test. This is the class-files analysis.

File: src/test/kotlin/com/autonomousapps/internal/ClassSetReaderTest.kt [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+        // When
+        val actualCore = ClassSetReader(
```
Comment (autonomousapps) [inline src/test/kotlin/com/autonomousapps/internal/ClassSetReaderTest.kt]: should be `actualApp`

File: src/test/kotlin/com/autonomousapps/internal/ClassSetReaderTest.kt [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+            classes = shelter.app.classesDir().walkTopDown().filter { it.isFile }.toSet(),
+            layouts = emptySet(),
+            kaptJavaSource = emptySet()
+        ).analyze()
+
+        // Then
+        val expectedCore = shelter.app.classReferences()
```
Comment (autonomousapps) [inline src/test/kotlin/com/autonomousapps/internal/ClassSetReaderTest.kt]: `expectedApp`

**Comment by stephanenicolas** [comment]
comments are addressed @autonomousapps

**Pull request merged**